docs/FAQ/FAQ-Wallet.md

```diff
@@ -1,133 +1,77 @@
 
-Enter following commands in Electrum console:
+### The gap limit problem
 
-```
- wallet.change_gap_limit(100)
- wallet.storage.write()
-```
-Restart your Electrum and verify that the newly set gap limit is correct by entering:
+Majority of third party wallets are [SPV wallets](https://en.bitcoinwiki.org/wiki/Simplified_Payment_Verification),share a node between many users. To  prevent performance issues, SPV wallets limit the amount of addresses without balance they follow on the blockchain. BTCPay Server generates a new address for every invoice. 
 
-`wallet.gap_limit`
+With above in mind, after BTCPay Server generates 20 consecutive unpaid invoices, the external wallet stops fetching the transactions, assuming no new transactions occurred. Once 21st, 22nd, etc invoices are paid, your external wallet won't show them.
 
-There's no good answer to how much you should set the gap limit to. Most merchants set 100-200. If you're a big merchants with high transaction volume, you can try with an even higher gap limit.
+On the other hand, internal BTCPay Server wallet is a full-node reliant wallet. It does not relay on a third-party and is able to always show a correct balance.
 
-Be aware that :
+### The gap limit solution
 
-* A higher gap limit may slow down the performance of your wallet
-* Not all wallets support the increased gap limit. If you import an Electrum recovery seed into another wallet, you may not see all the funds again.
+It's not easy to solve the gap limit problem. You have two options:
 
-When an invoice is created in BTCPay, it does it for all coins you have setup. You may want to increase the gap limit for altcoins as well in their supported wallets.
+1. Increase the gap limit in your external wallet
+2. Use internal BTCPay Server wallet
 
-If you do not see your funds yet, you may have set up your derivation scheme incorrectly.
+#### 1. Increasing the gap limit
 
-## Why is my Ledger not detected by BTCPay Server?
+If your wallets supports configurable gap-limit, the easy fix is to increase it. However, majority of wallets do not allow this. 
 
-Check that you are running the Ledger app with a version equal or above 1.2.4. Make sure to use Google Chrome. Plug your Ledger Nano S into your PC. Enter the PIN, and select the coin wallet app, for example > Bitcoin. Try refreshing the browser.
+The only wallets that allow configurable gap-limit value are [Electrum](/ElectrumWallet.md) and [Wasabi](/WasabiWallet.md)
 
-If the problem persists, contact the owner of the BTCPay Server: The reverse proxy of the BTCPay server hosting BTCPay might not support Websockets.
+If you'd like to use an external wallet to manage the funds, we recommend that perform a wallet recovery with those two wallets.
```

Comment:
I don't understand the meaning behind this sentence.

Do you mean that we should use Electrum or Wasabi and make a recovery of the wallet used ? Unclear.

Reply:
No idea what I did there ,fixing.